### PR TITLE
[JA] Change Japanese language to "ja"

### LINKED ISSUE #1322: Change Japanese language code to "ja"
Actually jp is country code.

It is not conflict with other langs but, still wrong.

- set redirect_from 
- rename folders 
- modify some settings.

Note that #1255 has effected by this changes.

related #1109 

ref: https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes

### PR DESCRIPTION
**Pull request by marocchino**
close #1322 

@elixirschool/japanese  review please. :)

### CODE REVIEW

File: _data/interlang.yml [hunk 1]
```diff
@@ -1,27 +1,27 @@
...
+#  "no":
+#  pl:
+#  pt:
+#  ru:
+#  sk:
+#  ta:
+#  th:
+#  tr:
+#  uk:
+#  vi:
```
Comment (eksperimental) [inline _data/interlang.yml]: the trailing space from all these lines should not be removed.

Comment (eksperimental) [reply _data/interlang.yml]: other than this, this PR looks good to me

Comment (marocchino) [reply _data/interlang.yml]: 🙆‍♂️ f4371af

**Comment by doomspork** [comment]
Thank you @marocchino, @riseshia, and @eksperimental ❤️

**Pull request merged**